packages/vue-instantsearch/src/util/pragma.js

```diff
@@ -0,0 +1,22 @@
+import { h, Fragment } from 'vue';
+
+export const createElement = (tag, props, children) => {
```

Comment:
is this one exactly the same for vue 2 and vue 3? maybe it's simpler if the it's written in two different files. If it is actually the same there's no issue and it looks good